2021-04-16 16:27:24,073 DEBUG    root            Loaded Command Group: ['gcloud', 'container']
2021-04-16 16:27:24,101 DEBUG    root            Loaded Command Group: ['gcloud', 'container', 'clusters']
2021-04-16 16:27:24,105 DEBUG    root            Loaded Command Group: ['gcloud', 'container', 'clusters', 'get_credentials']
2021-04-16 16:27:24,132 DEBUG    google.auth.transport.requests Making request: GET http://metadata.google.internal/computeMetadata/v1/instance/service-accounts/senmioxl@gmail.com/?recursive=true
2021-04-16 16:27:24,138 DEBUG    urllib3.connectionpool Starting new HTTP connection (1): metadata.google.internal:80
2021-04-16 16:27:24,142 DEBUG    urllib3.connectionpool http://metadata.google.internal:80 "GET /computeMetadata/v1/instance/service-accounts/senmioxl@gmail.com/?recursive=true HTTP/1.1" 200 None
2021-04-16 16:27:24,143 DEBUG    google.auth.transport.requests Making request: GET http://metadata.google.internal/computeMetadata/v1/instance/service-accounts/senmioxl@gmail.com/token
2021-04-16 16:27:24,147 DEBUG    urllib3.connectionpool Resetting dropped connection: metadata.google.internal
2021-04-16 16:27:24,150 DEBUG    urllib3.connectionpool http://metadata.google.internal:80 "GET /computeMetadata/v1/instance/service-accounts/senmioxl@gmail.com/token HTTP/1.1" 200 None
2021-04-16 16:27:24,165 DEBUG    google.auth.transport.requests Making request: GET http://metadata.google.internal/computeMetadata/v1/instance/service-accounts/senmioxl@gmail.com/?recursive=true
2021-04-16 16:27:24,168 DEBUG    urllib3.connectionpool Starting new HTTP connection (1): metadata.google.internal:80
2021-04-16 16:27:24,171 DEBUG    urllib3.connectionpool http://metadata.google.internal:80 "GET /computeMetadata/v1/instance/service-accounts/senmioxl@gmail.com/?recursive=true HTTP/1.1" 200 None
2021-04-16 16:27:24,172 DEBUG    google.auth.transport.requests Making request: GET http://metadata.google.internal/computeMetadata/v1/instance/service-accounts/senmioxl@gmail.com/token
2021-04-16 16:27:24,174 DEBUG    urllib3.connectionpool Resetting dropped connection: metadata.google.internal
2021-04-16 16:27:24,177 DEBUG    urllib3.connectionpool http://metadata.google.internal:80 "GET /computeMetadata/v1/instance/service-accounts/senmioxl@gmail.com/token HTTP/1.1" 200 None
2021-04-16 16:27:24,236 DEBUG    root            Running [gcloud.container.clusters.get-credentials] with arguments: [--project: "cohesive-slate-308002", --quiet: "True", --zone: "us-east1-b", NAME: "richard"]
2021-04-16 16:27:24,250 INFO     ___FILE_ONLY___ Fetching cluster endpoint and auth data.

2021-04-16 16:27:24,574 DEBUG    root            Saved kubeconfig to /home/senmioxl/.kube/config
2021-04-16 16:27:24,574 INFO     ___FILE_ONLY___ kubeconfig entry generated for richard.

2021-04-16 16:27:24,577 INFO     root            Display format: "default"
2021-04-16 16:27:24,577 DEBUG    root            SDK update checks are disabled.
